features/claims/litigator/scheme_nine/transfer_claim_draft_submit.feature

```diff
@@ -109,3 +109,85 @@ Feature: Litigator partially fills out a draft transfer claim, then later edits
     When I click View your claims
     Then I should be on the your claims page
     And Claim 'A20161234' should be listed with a status of 'Submitted' and a claimed amount of '£660.32'
+
+  @fee_calc_vcr
+  Scenario: I create a transfer claim for an Elected Case Not Proceeded
+
+    Given I am a signed in litigator
+    And My provider has supplier numbers
+    And I am on the 'Your claims' page
+    And I click 'Start a claim'
+    And I select the fee scheme 'Litigator transfer fee'
+    Then I should be on the litigator new transfer claim page
+    And I should see a page title "Enter fees for litigator transfer fees claim"
+
+    Then I choose the litigator type option 'New'
+    And I choose the elected case option 'Yes'
+    And I select the transfer stage 'Before trial transfer'
+    And I enter the transfer date '2015-05-21'
+
+    And I click "Continue" in the claim form
+
+    When I choose the supplier number '1A222Z'
+    And I select the court 'Blackfriars'
+    And I enter a case number of 'A20161234'
+    And I enter the case concluded date
+
+    And I should see a page title "Enter case details for litigator transfer fees claim"
```

Comment:
(nit) Would it make sense for this line to be the first one after 'Continue' on line 129, rather than just before the next 'Continue'?